src/main/java/com/robomwm/prettysimpleshop/shop/ShopListener.java

```diff
@@ -200,8 +201,13 @@ private void onOpenInventory(InventoryOpenEvent event)
     {
         if (event.getPlayer().getType() != EntityType.PLAYER)
             return;
-        if (event.getInventory().getLocation() == null)
-            return;
+        try {
+        	if (event.getInventory().getLocation() == null)
+        		return;
+        }
+        catch (NullPointerException e) {
```

Comment:
Should probably add a code comment stating this is a spigot/custom inventory bug. Perhaps reference whatever bug report when you make it.

Reply:
Also, did you indent with tabs or something? Spaces please.